apcd-cms/src/apps/view_users/views.py

```diff
@@ -19,22 +19,6 @@ def dispatch(self, request, *args, **kwargs):
     def get_context_data(self, user_content=user_content, *args, **kwargs):
         context = super(ViewUsersTable, self).get_context_data(*args, **kwargs)
 
-        def _set_user(usr):
```

Comment:
Why was this function removed but not its uses?

Reply:
I was a bit confused myself. But I'm pretty sure I haven't deleted anything that should be here, this time. Thank you for testing again if you get a chance.